addons/sale/models/account_move_line.py

```diff
@@ -181,7 +159,7 @@ def _sale_prepare_sale_line_values(self, order, price):
 
         return {
             'order_id': order.id,
-            'name': self.name,
+            'name': self.product_id.name,
```

Comment:
Why? 😇 🤔

Reply:
I think it is what we agreed on with aju. Right @deneuvillem ?

Reply:
Indeed, and this method is only used in the context of the reinvoicing flow, which we had to change some stuff to make it work with project. So it should not impact other sale flows

Reply:
While `self.name` and `self.product_id.name` are the same in most cases, it's not always true.
Why is this change needed, I don't understand the explanation "we had to change stuff"

Reply:
Actually we can leave it as `self.name`